src/home/LandingPage.js

```diff
@@ -3,8 +3,12 @@ import { Link } from 'react-router-dom';
 import Button from '@mui/material/Button';
 import Typography from '@mui/material/Typography';
 import Box from '@mui/material/Box';
+import Carousel from 'react-bootstrap/Carousel';
```

Comment:
NIT: (Nits are generally my personal opinion/suggestions). Most users don't bother scrolling through carousels. Prefer to display info vertically, especially since the landing page is bare.